appcell-resources/js/socialgraph.js

```diff
@@ -197,21 +195,23 @@ sg.dispExtCellLinkRelation = function(json, extUrl) {
   })
 
   if (results.length === 0) {
-      var dispName = sg.getName(extUrl);
-      var description = "";
-      var imageSrc = cm.notImage;
-      var extRelID = "dvExtCellRelList";
+      var profObj = {
+          DisplayName: sg.getName(extUrl),
+          Description: "",
+          Image: cm.notImage
+      }
+      var extRelObj = {ID: "dvExtCellRelList"};
```

Comment:
When you assign "dvExtCellRelList" here, would you be able to get it back in line 227 & 230

Reply:
This element is the default section, therore, the ID is always "dvExtCellRelList".